install_other_linux/tutorial.md

```diff
@@ -8,30 +8,28 @@ Debian, Fedora, Arch and Gentoo.
 ## Debian
 
 [Gazebo in Debian](https://packages.debian.org/source/sid/gazebo) is available
-as an official package since stretch. Debian Sid (the Gazebo team is the official
+as an official package since Stretch. Debian Sid (the Gazebo team is the official
 maintainer in Debian) usually hosts the latest gazebo release. Depending on the
 Debian version the version of Gazebo available is different:
 
  * Debian Sid is usually hosting one of the latest releases
  * Debian Buster: gazebo-9.6.0
- * Debian Stretch: gazebo-7.3.1
+ * Debian Bullseye: gazebo-11.1.0
 
-1. Install Gazebo9 (on Buster)
+1. Install Gazebo11 (on Bullseye)
 
-        sudo apt-get install gazebo9
+        sudo apt-get install gazebo11
         # For developers that works on top of Gazebo, one extra package
-        sudo apt-get install libgazebo9-dev
+        sudo apt-get install libgazebo11-dev
 
 ## Fedora
 
 [Gazebo in Fedora](https://apps.fedoraproject.org/packages/gazebo) is available
-as an official package (the maintainer is Rich Mattes). Depending on the Fedora
-version, the version of Gazebo available is different:
+as an official package.
 
- * Rawhide is usually hosting one of the latest releases
- * Fedora 31: gazebo-9.x
- * Fedora 30: gazebo-9.x
- * Fedora 29: gazebo-8.x
+ * Rawhide: gazebo-10
+ * Fedora 35: gazebo-10.x
+ * Fedora 36: gazebo-10.x
```

Comment:
Gazebo 10 is EOL. If there's no Gazebo 9 or 11 on these platforms, should we just remove them?

Reply:
Although Gazebo10 is EOL by our side (upstream) the distributions are free to keep shipping unsupported versions until they have a better alternative (i.e: In Debian we still have ogre-1.9).

I think it does not hurt to mention that here that Fedora is in version 10, maybe could help people to see that there is a need to update and someone contribute to Fedora to update it.